src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -195,7 +203,8 @@
                       Height="32"
                       Margin="4,0"
                       Padding="0"
-                      BackgroundSizing="OuterBorderEdge">
+                      BackgroundSizing="OuterBorderEdge"
+                      Click="CaseSensitivityButtonClicked">

```

Comment:
_technically,_ the way to do this kind of thing is to have a bunch of data bindings and property changed callbacks. It's "right," but it's not "fun" and it confers no benefit except that of clarity... so I am not going to block over it!